Dockerfile

```diff
@@ -0,0 +1,63 @@
+#
+# Dockerfile for building coordinator node images
+#
+
+FROM openjdk:8u312-jre-slim as base
+
+RUN apt update -qq \
+    && apt install iproute2 libaio1 -y \
+    && apt autoremove --yes \
+    && apt clean all \
+    && rm -rf /var/lib/apt/lists/*
+
+# CQL
+EXPOSE 9042
+
+# GraphQL
+EXPOSE 8080
+
+# Auth
+EXPOSE 8081
+
+# Health
+EXPOSE 8084
+
+# we use a script to build a directory with contents of ./stargate-lib, omitting all persistence jars
+# we can then add the proper set of persistence jars to each of the images below
+ARG LIBDIR
+COPY ${LIBDIR} /stargate-lib/
```

Comment:
Wouldn't this copy all jars, including persistence ones?

Reply:
Seems to me as well.. Ahh OK, seen the trick in the build file.. Why not simply going for:

```
RUN rm /stargate-lib/persistence*.jar
```

Reply:
@ivansenic I originally had a step in the image creation process that copied in all of the jar files and then removed the undesired ones but @tatu-at-datastax pointed out that this would be unnecessarily bloat the image since the files are still technically there but just don't appear in the directory. Plus it s a bit more complex than removing all `persistence*.jar` files, there are actually ones you want to keep per image.